installer/scripts/service_control

```diff
@@ -159,16 +349,22 @@ start_omsagent()
         elif [ -x /bin/systemctl ]; then
             /bin/systemctl start $OMSAGENT_WS
         else
-            echo "Unrecognized service controller to start OMS Agent service" 1>&2
+            echo "ERROR:  Unrecognized service controller to start OMS Agent service" 1>&2
             exit 1
```

Comment:
We can discuss this tomorrow, but this should be assigned a specific error code.